src/devices/LidarLiteV3/SystemStatus.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+
+namespace Iot.Device.LidarLiteV3
+{
+    /// <summary>
+    /// System Status
+    /// </summary>
+    [Flags]
+    public enum SystemStatus : byte {
```

Comment:
I don't think there is any value with this being public. If there is some utility here I'm missing I'd rather expose it as a separate method than making this public

Reply:
As mention earlier, I can at least see `ProcessError` error being useful, but I see your point in making the device simplier to use.  By not exposing it, the only way for the user to access it is by modifying the code.  I think what we really need is a higher-level device class, and a lower level one that exposes more of the device if they need it?

Reply:
We don't have clear guidelines about this yet so I'm fine with both approaches, I usually prefer simpler to use since for me hardware itself is a low level abstraction